cursorless-talon/src/actions/get_text.py

```diff
@@ -1,24 +1,48 @@
-from typing import Optional
-
-from talon import actions
+from talon import Module, actions
 
 from ..targets.target_types import CursorlessTarget
 
+mod = Module()
+
+
+@mod.action_class
+class Actions:
+    def cursorless_get_text(
+        target: CursorlessTarget,
+        no_decorations: bool = False,
+    ) -> str:
+        """Get target text"""
+        return cursorless_get_text(
+            target,
+            no_decorations=no_decorations,
+            ensure_single_target=True,
+        )[0]
 
-def cursorless_get_text_action(
+    def cursorless_get_text_list(
+        target: CursorlessTarget,
+        no_decorations: bool = False,
+    ) -> list[str]:
+        """Get texts for multiple targets"""
+        return cursorless_get_text(
+            target,
+            no_decorations=no_decorations,
+            ensure_single_target=False,
+        )
+
+
+def cursorless_get_text(
     target: CursorlessTarget,
     *,
-    show_decorations: Optional[bool] = None,
-    ensure_single_target: Optional[bool] = None,
+    no_decorations: bool,
+    ensure_single_target: bool,
 ) -> list[str]:
     """Get target texts"""
     options: dict[str, bool] = {}
 
-    if show_decorations is not None:
-        options["showDecorations"] = show_decorations
-
-    if ensure_single_target is not None:
-        options["ensureSingleTarget"] = ensure_single_target
+    if no_decorations:
+        options["showDecorations"] = False
+    if ensure_single_target:
+        options["ensureSingleTarget"] = True
```

Comment:
seems a bit odd to use a boolean to determine whether to pass args. I would argue for either keeping them as they were originally, where we don't pass anything if nothing comes in, or always passing them. Silently dropping them based on current default behaviour feels like it's asking for trouble

Reply:
Sure